test/integration/visitor_can_add_item_to_cart_and_view_cart_test.rb

```diff
@@ -1,14 +1,77 @@
 require 'test_helper'
 
 class VisitorCanAddItemToCartAndViewCartTest < ActionDispatch::IntegrationTest
-  test 'visitor can add item to cart' do
-    skip
+  test 'visitor can add item to cart from index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.